tidb-cloud/config-s3-and-gcs-access.md

```diff
@@ -9,7 +9,26 @@ If your source data is stored in Amazon S3 or Google Cloud Storage (GCS) buckets
 
 ## Configure Amazon S3 access
 
-To allow TiDB Cloud to access the source data in your Amazon S3 bucket, take the following steps to configure the bucket access for TiDB Cloud and get the Role-ARN.
+To allow TiDB Cloud to access the source data in your Amazon S3 bucket, you need to configure the bucket access for TiDB Cloud. You can use either of the following methods to configure the bucket access:
+
+- Use an Amazon access key: use the access key of an IAM user to access your Amazon S3 bucket.
+- Use a Role ARN: use a Role ARN to access your Amazon S3 bucket.
+
+<SimpleTab>
+<div label="Access Key">
+
+It is recommended that you use an IAM user (instead of the AWS account root user) to create an access key.
```

Comment:
It is recommended that you  create a new IAM user (instead of the AWS account root user) to create  access keys.